docpipeline/generate.py

```diff
@@ -223,3 +223,9 @@ def build_new_docs(bucket_name, credentials):
         if new_name not in other_names:
             new_blobs.append(blob)
     build_blobs(new_blobs, credentials)
+
+def build_language_docs(bucket_name, language, credentials):
```

Comment:
Done. Added a test with given Python file, it will build for a Python language and check that the build works. It will then build for a different language, Go chosen for now, and check that it doesn't modify the Python files.